lib/taurus/qt/qtgui/taurusgui/taurusgui.py

```diff
@@ -25,7 +25,7 @@
 
 """This package provides the TaurusGui class"""
 
-__all__ = ["DockWidgetPanel", "TaurusGui"]
+__all__ = ["DockWidgetPanel", "TaurusGui", "main"]
```

Comment:
Is there any reason for exporting the "main" function?
In general we avoid it (we do not consider them public API), and entry points use a full path to access them